docs/functions/usage/index.md

```diff
@@ -103,3 +104,22 @@ function App() {
   // ...
 }
 ```
+
+## Region-specific Functions
+
+If you need to deploy Functions in a region other than the default one, modified statements need to be used. 
+
+```javascript
+import { firebase } from '@react-native-firebase/functions';
+  
+  // Emulator statement 
+  firebase.app().functions("region_name").useEmulator('localhost', 5001);
+
+  // definition statement 
+  functions.region("region_name").https.onCall(() => {});
```

Comment:
Definition where? Where is this documented upstream? Could use a pointer. I'm not familiar with a need to define a function in a react-native app, I'm only familiar with defining functions in separate modules that are built + installed by the firebase tools.

Reply:
You are right. I removed the addition.